README.md

```diff
@@ -25,6 +25,8 @@ By adding additional env variables the container can send a html request to spec
 
 Example for a simple deployment can be found in `example.yaml`. Depending on the cluster setup you have to grant yourself admin rights first: `kubectl create clusterrolebinding cluster-admin-binding   --clusterrole cluster-admin   --user $(gcloud config get-value account)`
 
+One can override the default directory that files are copied into using a special annotation that looks like "k8s-sidecar-target-directory: /path/to/target/directory" the sidecar will attempt to create this directory if it does not exist.
```

Comment:
Why do you want to use annotations and not labels?

Reply:
I don't feel strongly either way. annotations seem a bit better to me for an argument that is optional. I could be convinced otherwise.

Reply:
You are right, annotations should are better fitting for this use case.

Reply:
This is probably a major major reason to use an annotation for a folder name:
> kubernetes label name 63 character limit. As explained at https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ labels name and value have limit of 63 characters.